### DISCUSSION TITLE
No browsersync? And where's the compiler?

### DISCUSSION THREAD

**Discussion by currentcreative** [bootscore/bootscore#730] [General]
Coming from Understrap, I was impressed to see all the plugins included in Bootscore, but I was pretty surprised by the lack of commands needed to do different things like create a distributable version of the theme without npm modules and lack of browser sync.

• Are there no npm modules in this thing? I don't see any. How does it work?

• I'm confused where the compiler is. If I upload both of these themes to my server, am I going to be uploading a bunch of unnecessary compiler scripts to my production site?

• How can I get browsersync working on this child theme?

**Comment by crftwrk** [comment]
Theme uses scssphp and compiles source by itself everytime again when something in the .scss files has changed. There is no npm here. If you do not want to use scssphp compiler, you can simply disable it by a filter and use compiler of your choice. Everything about the compiler is written down here https://bootscore.me/documentation/theme/bootstrap-css-sass/.

> How can I get browsersync working on this child theme?

Not supported

**Reply by currentcreative** [reply]
OK, thanks for the reply. If anyone happens to see this who has successfully set up BrowserSync or some other type of live reloading for BootScore, please let me know how you did it.

**Reply by peterjrees** [reply]
+1 for this. I would love to try Bootscores in some of my projects but no live reloading is a deal-breaker for me.

**Reply by peterjrees** [reply]
I was able set up Browsersync on my Local WP development environment (using Bootscore child theme) by doing the following:

1. Install Browsersync with NPM
`npm install browser-sync --save-dev`

2. Add script to package.json (change url as needed)
```
"scripts": {
    "dev": "browser-sync start --proxy 'https://bootscore.local' --files 'assets/scss/*.scss, *.php, assets/js/*.js'"
  }
```
3. Run with
`npm run dev`

This will watch your SCSS, PHP, and JS files and reload automatically when any changes are saved.
